torch/autograd/profiler.py

```diff
@@ -101,55 +101,39 @@ def export_chrome_trace(self, path):
         Arguments:
             path (str): Path where the trace will be written.
         """
-        import json
+        import os
         with open(path, 'w') as f:
             chrome_events = []
             next_id = 0
+            # Use file IO over using json.dump since JSON dumping is very slow and
+            # this technique is proven to give a 4x speedup.
+            f.write("[")
             for evt in self:
-                chrome_events.append(dict(
-                    name=evt.name,
-                    ph='X',
-                    ts=evt.cpu_interval.start,
-                    dur=evt.cpu_interval.elapsed_us(),
-                    tid=evt.thread,
-                    pid='CPU functions',
-                    args={},
-                ))
+                f.write("{\"name\": \"%s\", \"ph\": \"X\", \"ts\": %s, " \
```

Comment:
These backslashes are not needed and are causing linting failures. JSON needs `"` but Python is happy with either single or double quotes, so use:
```
'add "some" double quotations "inside" single quotes'
```